core/lib/l1_contract_interface/src/i_executor/methods/commit_batches.rs

```diff
@@ -9,6 +9,7 @@ use crate::{
     Tokenizable, Tokenize,
 };
 
+const SUPPORTED_ENCODING_VERSION: [u8; 1] = [0];
```

Comment:
A cleaner way is to have `VERSION` as a u8 constant and have `vec![SUPPORTED_ENCODING_VERSION]` created with it on the spot